scripts/procdockerstatsd

```diff
@@ -168,6 +168,30 @@ class ProcDockerStats(daemon_base.DaemonBase):
                 cmd = row.get('CMD')
                 self.update_state_db(value, 'CMD', cmd)
 
+    def update_procfipsstats_command(self):
+        fips_enable_flag = '0'
+        current_enforce_state = '0'
+        next_enforce_state = '0'
+        if os.path.isfile('/etc/fips/fips_enable.flag'):
```

Comment:
>/etc/fips/fips_enable.flag [](http://example.com/codeflow?start=27&length=26)

ConfigDB is the ground truth of config, why you need to monitor a file?

Reply:
Removed the file, used the StateDB instead, thanks.